modules/authorization-manager/src/main/java/org/opencastproject/authorization/xacml/manager/endpoint/AbstractAclServiceRestEndpoint.java

```diff
@@ -96,10 +96,19 @@
   @GET
   @Path("/acl/{aclId}")
   @Produces(MediaType.APPLICATION_JSON)
-  @RestQuery(name = "getacl", description = "Return the ACL by the given id", returnDescription = "Return the ACL by the given id", pathParameters = { @RestParameter(name = "aclId", isRequired = true, description = "The ACL identifier", type = INTEGER) }, responses = {
+  @RestQuery(
+      name = "getacl",
+      description = "Return the ACL by the given id",
+      returnDescription = "Return the ACL by the given id",
+      pathParameters = {
+          @RestParameter(name = "aclId", isRequired = true, description = "The ACL identifier", type = INTEGER)
```

Comment:
This formatting doesn't match that in #2273, specifically [here](https://github.com/opencast/opencast/pull/2273/files#diff-f57bf6edb4770a3a203d7f49b0d9b7c9e2f6a9b4ba4e13f1393898721b880d5fR90).  I'm fine with either, with a mild preference for this format.  That being said, my *biggest* preference is that whatever we have is something that an IDE spits out when configured properly.

Reply:
I tried configuring IntelliJ in its settings UI and I couldn't make it produce this formatting. There aren't a lot of settings regarding annotations, unfortunately.